std/math.d

```diff
@@ -7525,10 +7525,13 @@ if (isFloatingPoint!(F) && isIntegral!(G))
 typeof(Unqual!(F).init * Unqual!(G).init) pow(F, G)(F x, G n) @nogc @trusted pure nothrow
```

Comment:
The documentation above will need to be corrected too, if this is to be accepted.